src/components/Banner/Delegation/DelegationBanner.tsx

```diff
@@ -0,0 +1,51 @@
+import './DelegationBanner.css'
+
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import React, { useEffect } from 'react'
+
+import Delegate from '../../Icon/Delegate'
+import Banner, { BannerProps, BannerState, ShowBanner } from '../Banner'
+
+type DelegationBannerProps = {
+  bannerState: BannerState
+  active?: boolean
+}
+
+const HIDE_DELEGATE_BANNER_KEY = 'org.decentraland.governance.delegate_banner.hide'
+
+const button = (label: string) => {
+  return (
+    <Button className="Banner__Button" primary size="small" href={process.env.GATSBY_DELEGATE_APPLICATION_URL || ''}>
+      {label}
+    </Button>
+  )
+}
+
+function DelegationBanner({ active, bannerState }: DelegationBannerProps) {
+  const t = useFormatMessage()
+  const { setShowBanner } = bannerState
```

Comment:
I'm thinking that maybe it'd be better if the component can handle it's state internally rather than receiving a state. 

It could receive just the initial configuration (if needed), but it's state should live inside of it (or the Banner component)